hardware/plugins/PluginTransports.cpp

```diff
@@ -92,7 +90,7 @@ namespace Plugins {
 		if (!err)
 		{
 			boost::asio::ip::tcp::endpoint endpoint = *endpoint_iterator;
-			m_Socket->async_connect(endpoint, boost::bind(&CPluginTransportTCP::handleAsyncConnect, this, boost::asio::placeholders::error, ++endpoint_iterator));
+			m_Socket->async_connect(endpoint, [this, endpoint_iterator](const boost::system::error_code &err) mutable { handleAsyncConnect(err, ++endpoint_iterator); });
```

Comment:
Just a side-note, there's a much nicer way to handle connections nowadays: https://www.boost.org/doc/libs/1_74_0/doc/html/boost_asio/reference/async_connect/overload1.html

Definitely not in scope for this PR though